docs/setup/pick-right-solution.md

```diff
@@ -60,6 +60,8 @@ a Kubernetes cluster from scratch.
 
 * [IBM Bluemix Container Service](https://console.ng.bluemix.net/docs/containers/container_index.html) offers managed Kubernetes clusters. Currently in beta.
 
+* [GiantSwarm.io](https://giantswarm.io/product/) offers managed Kubernetes clusters in own datacenter, on-premises or in cloud.
```

Comment:
`in own datacenter, on-premises or in cloud.`-> `in their own datacenter, on-premises, or on public clouds.`